api/models/User.js

```diff
@@ -26,7 +26,7 @@ var User = {
     // Method to return JSON to the API
     toJSON: function() {
       var obj = this.toObject();
-      delete obj.passports;
+      // delete obj.passports;
```

Comment:
I don't know but my guess would be that it wouldn't since all the passports will be associated with the User model. Since I can't think of a case where a passport would be shared amongst two users I think we're safe.

That being said, the goal for this was to get the token to the front end for use by the editor. Do you think setting this as a cookie would be better instead of including it in the JSON response?